x/claim/keeper/mission.go

```diff
@@ -88,6 +72,62 @@ func (k Keeper) CompleteMission(ctx sdk.Context, missionID uint64, address strin
 	}
 	claimRecord.CompletedMissions = append(claimRecord.CompletedMissions, missionID)
 
+	k.SetClaimRecord(ctx, claimRecord)
+
+	err := ctx.EventManager().EmitTypedEvent(&types.EventMissionCompleted{
+		MissionID: missionID,
+		Address:   address,
+	})
+	if err != nil {
+		return err
+	}
+
+	// try to claim the mission if airdrop start is reached
+	airdropStart := k.AirdropStart(ctx)
+	if !ctx.BlockTime().Before(airdropStart) {
```

Comment:
There isn't an `After` method?

Reply:
if we check if it is not before we can cove the corner case the timestamp can be after or also equal

Reply:
It's a matter of a nano second isn't it?